From ed2c42a0457bb7b8b342b025fad247262196e9c6 Mon Sep 17 00:00:00 2001
From: Gao Pan <b54642@freescale.com>
Date: Wed, 16 Dec 2015 16:38:34 +0800
Subject: [PATCH 0774/5242] MLK-12021 ARM: dts: add sim1 support for imx7d-sdb
 board

commit  cad15449e6bed6d92573471bb715235fced293ac from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable sim1 in imx7d-sdb.dts. To avoid the pin confict with epdc,
disable sim1 in imx7d-sdb-epdc.dts.

Signed-off-by: Gao Pan <B54642@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-epdc.dts |    5 +++++
 arch/arm/boot/dts/imx7d-sdb.dts      |    2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dts b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
index fc2c656..9eb5d83 100644
--- a/arch/arm/boot/dts/imx7d-sdb-epdc.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dts
@@ -31,3 +31,8 @@
 &sii902x {
 	status = "disabled";
 };
+
+&sim1 {
+	status = "disabled";
+};
+
diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index c002d67..421cfee 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -634,7 +634,7 @@
 	pinctrl-0 = <&pinctrl_sim1_1>;
 	port = <0>;
 	sven_low_active;
-	status = "disabled";
+	status = "okay";
 };
 
 
-- 
1.7.9.5

